src/windows/ContactProxy.js

```diff
@@ -242,7 +242,9 @@ module.exports = {
         pickRequest.done(function (contact) {
             // if contact was not picked
             if (!contact) {
-                fail && fail(new Error("User did not pick a contact."));
```

Comment:
This looks good.  The implied check for `fail != null` here was unnecessary as ALL calls to exec proxies have valid success/error function passed in.